test/integration/migrations_test.rb

```diff
@@ -266,6 +266,13 @@
         migration.fetch(:filename).must_include("20150610141017")
       end
 
+      it "ignore files starting with . during migration" do
+        connection = Sequel.connect(@uri)
+        migrations  = connection[:schema_migrations].to_a.select { |migration| migration.fetch(:filename).start_with?('.') }
```

Comment:
I think you should change this var name, because you are selecting file starting with `.` which they aren't migrations.
You could name `hidden_files`, for example, because we are getting that kind of files.

On more thing, fix indentation for `=`